pysmt/smtlib/commands.py

```diff
@@ -66,6 +67,7 @@
     GET_PROOF,
     GET_UNSAT_CORE,
     GET_INFO,
+    GET_INTERPOLANTS,
```

Comment:
This is not really part of SMTLIB, as the extension is not part of the standard. What if we move this to a new list called `EXTENSIONS` and then define `ALL_COMMANDS` below as `SMT_LIB_2_5 + EXTENSIONS` ?